projects/npm-tools/packages/npm-scripts/src/utils/parseBnd.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const fs = require('fs');
+
+/**
+ *
+ */
+function parseBnd(filePath = './bnd.bnd') {
+	const content = fs.readFileSync(filePath).toString();
+	const lines = content.replace(/\\\n/g, ' ').split('\n');
+
+	return lines.reduce((entries, line) => {
+		line = line.trim();
+
+		if (line === '') {
+			return entries;
+		}
+
+		const i = line.indexOf(':');
+		const key = line.substring(0, i);
+		const value = line.substring(i + 2);
+
+		entries[key] = value;
+
+		return entries;
```

Comment:
Should we handle line continuations (`\`)? It seems a bit dangerous to call the function `parseBnd` if it only produces a valid parse result for a subset of bnd files (ie. ones without properties that span multiple lines).

Reply:
We are handling them (here](https://github.com/liferay/liferay-frontend-projects/pull/252/files#diff-333307525d25dd06bb5a64f84cc557d8cb21337d0be2a934f1947bf107153e2eR22). Maybe I should add comments explaining what I'm doing, since regexps are so cryptical.

Reply:
Where it says "here" it is referring to line 22 (my github UI doesn't jump to the expected place when I click the link :roll_eyes: ).

Reply:
Ignore this — I now see you actually are handling line continuations.

Reply:
> Maybe I should add comments explaining what I'm doing

Nooo! It is better to live in the darkness of ignorance.